app/Services/Webpack.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace App\Services;
+
+class Webpack
+{
+    /**
+     * @var string
+     */
+    private $manifest_path;
```

Comment:
書けるものは極力docsを書きたくない。
```suggestion
    private string $manifest_path;
```

Reply:
後これは好みもあるのかも知れないけど、PHPでスネークケースまじ？と言う人種なので違和感があるけど、これは何記法って言うんだろう？
メソッドはロワーキャメル、クラスはアッパーキャメル、変数はスネークか。

Reply:
> PHPでスネークケース

おｋ。変数もキャメルケースにする。